games-puzzle/brainparty/brainparty-0.61.ebuild

```diff
@@ -30,19 +36,16 @@ src_prepare() {
 		-e '/-o brainparty/s/INCLUDES) /&$(LDFLAGS) /' \
 		Makefile || die
 	sed -i \
-		"/^int main(/ a\\\\tchdir(\"${GAMES_DATADIR}/${PN}\");\n" \
+		"/^int main(/ a\\\\tchdir(\"/usr/share/${PN}\");\n" \
 		main.cpp || die
-	epatch \
-		"${FILESDIR}"/${P}-savegame.patch \
-		"${FILESDIR}"/${P}-gcc49.patch
 
+	default
```

Comment:
I would rather put that first inside src_prepare, then do all these seds afterwards. That way (user) patches apply over vanilla source.